linera-sdk/src/contract/runtime.rs

```diff
@@ -276,8 +276,8 @@ where
     }
 
     /// Adds a new item to an event stream.
-    pub fn emit(&mut self, name: StreamName, key: &[u8], value: &[u8]) {
-        contract_wit::emit(&name.into(), key, value);
+    pub fn emit(&mut self, name: StreamName, value: &[u8]) -> u32 {
```

Comment:
Maybe add to the comment what's the `u32` returned.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3672/commits/9cf84642f6db6a6ef96b15601d7726d820f1a59b.